test/cpp/util/tls_test_utils.cc

```diff
@@ -94,5 +94,18 @@ void AsyncCertificateVerifier::WorkerThread(void* arg) {
   }
 }
 
+bool VerifiedRootCertSubjectVerifier::Verify(
+    TlsCustomVerificationCheckRequest* request,
+    std::function<void(grpc::Status)>, grpc::Status* sync_status) {
+  if (request->verified_root_cert_subject() != expected_subject_) {
+    *sync_status = grpc::Status(grpc::StatusCode::UNAUTHENTICATED,
+                                "VerifiedRootCertSubjectVerifier failed");
+  } else {
+    // *sync_status = grpc::Status(grpc::StatusCode::OK, "");
```

Comment:
nit: Remove the commented code, just in case you forgot.

Reply:
Good catch